web/groups/hlt.json

```
@@ -288,6 +290,11 @@
   "HLTHGCalDigisInRegionsProducer|": "HGCal",
   "HLTHGCalRecHitsInRegionsProducer|hltRechitInRegionsHGCAL": "E/Gamma",
   "HLTHGCalRecHitsInRegionsProducer|": "HGCal",
+  "HGCalSoARecHitsProducer@alpaka|hltHgcalSoARecHitsProducer": "HGCal|Alpaka",
```

In the Run 3 pies and in other modules here, we use `GPU` also for the alpaka-based modules, not explicitly `Alpaka`.
Do you explicitly want to use `Alpaka` ?

---

Can you leave the module name out, if it's not required to disambiguate the group ?
```suggestion
  "HGCalSoARecHitsProducer@alpaka|": "HGCal|Alpaka",
```

---

I have no strong preference. It reads weird reading GPU for Alpaka code executed on the CPU backend. Hence, my choice here.

---

To be consistent we should then change all alpaka-based modules to `Alpaka`.
But I would prefer to do it in a separate PR (and maybe after we drop the CUDA based modules form the release).

---

By the way, if you update this file, please update also `hlt_no_gpu.json`.
It should have all the same patterns and modules, but without the `GPU` sub-grouping.